lib/Doctrine/ODM/MongoDB/Id/AutoGenerator.php

```diff
@@ -9,6 +9,8 @@
 
 /**
  * AutoGenerator generates a native ObjectId
+ *
+ * @deprecated
```

Comment:
We try to add helpful details.
```suggestion
 * @deprecated use ObjectIdGenerator instead.
```

Reply:
Updated